doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,201 @@
+
+If the node does not find a network to join and it has either the Coordinator or the Router role, it can attempt to form a new network.
+If the node has the End Device role, its commissioning procedure stops here.
+
+The new device must be authenticated and authorized to become part of the network.
+For `Centralized security network`_, the authentication and authorization involves establishing a unique Trust Center Link Key (TCLK).
+
+Network Steering in the |NCS|
+   In the ZBOSS stack, Network Steering is started using the `ZB_BDB_NETWORK_STEERING`_ bit mask.
+
+   In the |NCS|, Network Steering for a node on a network can be started using the following additional options:
+
+   * When running the :ref:`Zigbee Coordinator sample <zigbee_network_coordinator_sample>`, you can press the associated button.
+   * If you are using the :ref:`lib_zigbee_shell` library in your application, you can use the ``bdb start`` command.
+
+.. _zigbee_commissioning_modes_nf:
+
+Network Formation
+=================
+
+The Network Formation mode is available only to Coordinators and Routers, because End Devices are not capable of forming a Zigbee network.
+
+.. figure:: /images/zigbee_commissioning_formation.svg
+   :alt: Zigbee Network Formation procedure
+
+   Zigbee Network Formation procedure
+
+In this mode, the node has not found a network to join during the Network Steering mode, and so it attempts to form a new network.
+Depending on the node role:
+
+* A Coordinator forms a centralized security network.
+* A Router forms a distributed security network.
+
+If Network Formation is successful, you can add nodes to the new network using `Network Steering`_.
+If unsuccessful, the commissioning procedure stops here.
+
+Network Formation in the |NCS|
+   In the ZBOSS stack, Network Formation is started using the `ZB_BDB_NETWORK_FORMATION`_ bit mask.
+
+   In the |NCS|, when using the :ref:`lib_zigbee_signal_handler` and the application implements the Zigbee Coordinator role, the Network Formation procedure is started right after the stack initialization.
+
+Centralized security network
+----------------------------
+
+The centralized security network is started by the Zigbee Coordinator node, which assumes the Trust Center functionality.
+As such, it is responsible for managing the security and authorization in the network.
+In the centralized security network, only the Trust Center can provide the network key to the joining device.
+This happens by using a link key specific to the joining device.
+The link key is exchanged with the Trust Center, which allows for encrypting packets exchanged between the Trust Center and the new node.
+This way, only the Trust Center and the new node can decrypt the packages exchanged between them.
+The link key value can be defined in one of the following ways:
+
+* Calculated based on the joining device's install code.
+* Predefined in the specification: ``"ZigBeeAlliance09"``.
```

Comment:
Looks like procedure of requesting TCLK is missing?

Reply:
@tomchy , do we need this procdure? :D

Reply:
Rephrased.

Reply:
Rephrased.